src/models/device-type.ts

```diff
@@ -434,8 +439,9 @@ const getDeviceTypeModel = function (deps: InjectedDependenciesParam) {
 		getInstructions: async (
 			deviceTypeSlugOrContract: string | Contract,
 		): Promise<Record<'Linux' | 'MacOS' | 'Windows', string[]> | string[]> => {
-			let contract: DeviceType['contract'];
+			let contract: Contract;
```

Comment:
```suggestion
			let contract: Contract | null = null;
```
since the `contract` JSON field returned by getBySlugOrName is nullable.
Otherwise eslint/TS might later start warning that the `?.` in `contract?.partials` isn't necessary.